src/archutils/Win32/GraphicsWindow.cpp

```diff
@@ -69,10 +69,10 @@ static LRESULT CALLBACK GraphicsWindow_WndProc( HWND hWnd, UINT msg, WPARAM wPar
 			if( !g_bHasFocus )
 			{
 				RString sName = GetNewWindow();
-				static set<RString> sLostFocusTo;
+				static std::set<RString> sLostFocusTo;
 				sLostFocusTo.insert( sName );
 				RString sStr;
-				for( set<RString>::const_iterator it = sLostFocusTo.begin(); it != sLostFocusTo.end(); ++it )
+				for( auto it = sLostFocusTo.begin(); it != sLostFocusTo.end(); ++it )
```

Comment:
Same here - should be `cbegin()` and `cend()` if you want the derived type to remain a `const_iterator`.